chapters/introduction.tex

```diff
@@ -72,6 +72,11 @@ \section{Some Definitions}\label{some-definitions}
 and components, and generation of connection equations from \lstinline!connect!-equations (basically, mapping the hierarchical structure of a model into a set of differential, algebraic and discrete equations together with the corresponding variable declarations and function definitions from the model).
 \end{definition}
 
+\begin{definition}[Simulation]\index{simulation}
```

Comment:
Good start, but I think it would serve us well if we first had proper definitions of _initialization problem_ and (say) _transient analysis_, allowing _simulation_ to be defined as the combination of the two.

It would also give us a chance to move the `\firstuse{initial value problem}` to this introductory chapter, instead of this ending up in an appendix.

Then, we define _transient analysis_ (or whatever term we prefer) here, there are also some places affected by the PR where we should use this term instead of making awkward constructs based on _simulation_.

Reply:
Now done.

Reply:
Nice.  I'll add a new comment above about the `\firstuse{initial value problem}`.  Resolving.